### PR TITLE
[Springt: 48] XD-2921 Update docs clarifying some Sqoop job issues

### PR DESCRIPTION
**Pull request by trisberg**
- Clarify the use of escape characters for options in the Sqoop job definition

- Clarify the use of the Hadoop config files

### CODE REVIEW

File: src/docs/asciidoc/Batch-Jobs.asciidoc [hunk 1]
```diff
@@ -725,6 +725,36 @@ job create sqoopComplexArgs1 --definition "sqoop --command=import --args='--tabl
 
 (For this example we have omitted the equal sign for the individual Sqoop arguments within the `--args` option. Either style works fine.)
 
+NOTE: If your Sqoop args use escape sequences (common when working with Hive data) then you should provid double back-slash characters when working with the XD Shell (this effectively escapes the escape character and only one back-slash will be passed on). Here is a brief example:
```
Comment (ericbottard) [inline src/docs/asciidoc/Batch-Jobs.asciidoc]: typo: provid

Comment (ericbottard) [inline src/docs/asciidoc/Batch-Jobs.asciidoc]: can refer to the section about quotes handling in the documentation

File: src/docs/asciidoc/Batch-Jobs.asciidoc [hunk 2]
```diff
@@ -725,6 +725,36 @@ job create sqoopComplexArgs1 --definition "sqoop --command=import --args='--tabl
...
+- `$XD_HOME/config/hadoop-site.xml` -- add a property entry:
+
+    <property>
+      <name>dfs.client.socket-timeout</name>
+      <value>20000</value>
+    </property>
+
+- `$XD_HOME/config/servers.yml` -- add a spring.hadoop.config entry:
+----
+---
```
Comment (ericbottard) [inline src/docs/asciidoc/Batch-Jobs.asciidoc]: Do we really want yml document delimiters here?

Comment (trisberg) [inline src/docs/asciidoc/Batch-Jobs.asciidoc]: I can remove them

File: src/docs/asciidoc/Batch-Jobs.asciidoc (hunk 1 above)
Comment (trisberg) [inline src/docs/asciidoc/Batch-Jobs.asciidoc]: Adding the xref

**Comment by trisberg** [comment]
Making changes and will push new squashed commit

**Comment by ericbottard** [comment]
Merged as d9e92c623616a3fcc58edc3ae3e5b4e80173b45b.

**Pull request closed without merging**